tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "jsx": "react",
     "types": ["node", "jasmine"],
     "downlevelIteration": true,
+    "target": "es2015",
```

Comment:
Why is this one necessary?

Reply:
We removed these when alex moved them into the BUILD level config. But then you get lots of complaints in your IDE about features not being supported, so I'm adding back in what's needed.

Reply:
IIRC this breaks something else (when I last tried it), but maybe I did something else wrong.  No problem.

Reply:
Seems to work so far, but we get a warning from the bazel build. I think that's better than a broken IDE...